modules/juce_audio_devices/native/juce_android_OpenSL.cpp

```diff
@@ -620,21 +643,26 @@ class OpenSLAudioIODevice  : public AudioIODevice,
                                                                    static_cast<SLuint32> (bufferList.numBuffers) };
             SLDataSink audioSink = { &bufferQueue, &pcmFormat };
 
-            const SLInterfaceID interfaceIDs[] = { *engine.SL_IID_ANDROIDSIMPLEBUFFERQUEUE };
+            const SLInterfaceID interfaceIDs[] = { *engine.SL_IID_ANDROIDSIMPLEBUFFERQUEUE,
+                *engine.SL_IID_ANDROIDCONFIGURATION };

```

Comment:
Ok, that's what it looked like also, just wanted to confirm. Good job finding this 👍